otcextensions/sdk/dms/v1/group.py

```diff
@@ -56,17 +55,17 @@ class Group(_base.Resource):
     #: *Type: int*
     available_deadletters = resource.Body('available_deadletters', type=int)
 
-    def create(self, session, group):
+    def create(self, session, *args, **kwargs):
         """create group"""
-        body = {"groups": [{'name': group}]}
+        body = {"groups": [{'name': self.name}]}
 
         request = self._prepare_request(requires_id=False,
                                         prepend_key=False)
 
         response = session.post(
```

Comment:
Can we add static delay for the following note?
_After you create a consumer group, it takes 1s to 3s to initialize the system. If you operate the queue
immediately after creating the consumer group, the consumption may fail. You are advised to perform
operations 3s later._

Reply:
Nope, we will not add delays on our side. It's task of the user (if there ever be one) to implement this in the code if necessary